### DISCUSSION TITLE
How to wrap text without spaces in the View or Content

### DISCUSSION THREAD

**Discussion by makarandkeer** [adobe/react-spectrum#3412] [Q&A]
I have scenario where read-only text inside View or Content should auto wrap if it goes beyond its fixed width. This work correctly of string has spaces in it. However in some edge cases where text does not have any spaces I wanted it to wrap the text as it is doing when used with TextArea. But with View and Content as shown in SS below text goes outside width of the component, 
Refer: https://codesandbox.io/s/react-spectrum-template-forked-iwfyvi?file=/src/App.js

![image](https://user-images.githubusercontent.com/32228809/185453441-e698b3c5-2ee2-4c51-8a4b-153d2ece3430.png)

**Comment by snowystinger** [comment]
View isn't really meant to handle that, it's meant to be a way to access Spectrum CSS variables. You can put a div inside the View and handle overflow and word-break/wrap however you please.
Content isn't meant to be used standalone, it's there to receive props on context from a parent, such as Dialog or ListItem.
